pyproject.toml

```diff
@@ -60,7 +60,6 @@ docs = [
     "towncrier==22.12.0",
     "sphinx-notfound-page==0.8.3",
 ]
-discord = ["discord-disnake"]
```

Comment:
By removing this we can re-optimize multiple workflows to install *all* dependencies again, and ignore this by default.

Don't worry, this is still declared due to pdm_build.py